tiptapy/extras.py

```diff
@@ -8,15 +8,19 @@ class FeatureImage(BaseNode):
     def inner_render(self, node) -> str:
         special_attrs_map = {'caption': 'figcaption'}
         attrs = node.get("attrs", {})
-        attrs_s = " ".join(f'{k}="{v}"'
-                           for k, v in attrs.items()
-                           if k not in special_attrs_map
-                           )
-        html = f"<picture><img {attrs_s}><picture>"
-        for attr in special_attrs_map:
-            if attr in attrs:
-                tag = special_attrs_map[attr]
-                html += f"<{tag}> {attrs[attr]} </{tag}>"
+        attrs_s = " ".join(
+            f'{k}="{v}"'
+            for k, v in attrs.items()
+            if k not in special_attrs_map and
+            len(attrs[k]) != 0
```

Comment:
`and attrs[k]`

Reply:
@shon  I was thinking something like this 
```
attrs_s = " ".join(
            f'{k}="{v}"'
            for k, v in attrs.items()
            if k not in special_attrs_map and v
        )
```